### PR TITLE
release updates for 1.3.4

### PR DESCRIPTION
**Pull request by JWittmeyer**
- https://github.com/code-kern-ai/refinery-ui/pull/70
- https://github.com/code-kern-ai/refinery-gateway/pull/61
- https://github.com/code-kern-ai/refinery-submodule-model/pull/18

### CODE REVIEW

File: util/doc_ock.py [hunk 1]
```diff
@@ -22,15 +24,31 @@ def register_user(user: models.User):
...
+            {"eventName": event.event_name(), **event.__dict__},
+            caller_dict,
         )
         service_requests.post_call_or_raise(url, event.__dict__)
     except Exception:
         print(traceback.format_exc(), flush=True)
+
+
+def __get_caller_data() -> Dict[str, Union[str, int]]:
+    # not this, not the parent (post_event) but the one before
```
Comment (SirDegraf) [inline util/doc_ock.py]: not sure what this comment explains to me, what is meant with not this

File: util/user_activity.py [hunk 2]
```diff
@@ -5,28 +5,16 @@
...
-        "caller_lineno": parent.lineno,
-    }
-
-
-def add_user_activity_entry(user_id: str, activity: Any) -> None:
-    caller_dict = get_caller_data()
+def add_user_activity_entry(
+    user_id: str, activity: Any, caller_dict: Dict[str, Union[str, int]]
+) -> None:
+    # caller_dict = get_caller_data()
```
Comment (SirDegraf) [inline util/user_activity.py]: can comment be removed?

File: util/doc_ock.py (hunk 1 above)
Comment (JWittmeyer) [reply util/doc_ock.py]: we talked about this the last time. Its in regards to the stack trace index to explain the number used. 
but I can remove it

File: util/user_activity.py (hunk 2 above)
Comment (JWittmeyer) [reply util/user_activity.py]: will be removed

File: util/doc_ock.py (hunk 1 above)
Comment (JWittmeyer) [reply util/doc_ock.py]: changed the comment

**Review by lumburovskalina — APPROVED** [review]

**Pull request merged**